generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.ts.ejs

```diff
@@ -62,10 +68,12 @@ export class PasswordResetFinishComponent implements OnInit, AfterViewInit {
     finishReset() {
         this.doNotMatch = null;
         this.error = null;
-        if (this.resetAccount.password !== this.confirmPassword) {
+        const password = this.passwordForm.get(['newPassword']).value
+        const confirmPassword = this.passwordForm.get(['confirmPassword']).value
+        if (password !== confirmPassword) {
```

Comment:
an alternative could be to write custom validator function

Reply:
Yes, but I wouldn't change the original code so much